### PR TITLE
Updating design tokens package and shadow values with new tokens

### LINKED ISSUE #15204: Update shadows in extension with design token shadows
### Description
Update design token library @metamask/design-tokens to v`1.8` and update all instances of box shadows to use design token css variables

### Technical Details
- update `@metamask/design-tokens` package to v`1.7`
- update all instances of `box-shadow`, `text-shadow` and `filter: drop-shadow(` to use appropriate css shadow design tokens

### Acceptance Criteria
- All box-shadows are using a combination of the shadow tokens

### PR DESCRIPTION
**Pull request by georgewrmarshall**
## Explanation
Currently, shadows are not standardized across the app

This is a problem because it creates inconsistent UI elevation patterns. The current shadows are also not optimized for darkmode and sometimes don't contrast very well against a dark background making them hard to see.

In order to solve this problem, this pull request updates all instances of shadows in the css to use some combination of the shadow css design tokens

## More Information

* Fixes #15204 
- Going to add documentaiton in another issue so it can be reviewed more thoroughly  https://github.com/MetaMask/metamask-extension/issues/15299

## Screenshots/Screencaps

Because this PR stretches across numerous components I will add the before / after visual changes on each instance of box-shadow code to make it easier for review

## Manual Testing Steps

- Go to the storybook build on this PR
- Hopefully each update has a story which will make it easy to view the shadow changes
- Use the story for each component to see the shadow changes

Alternatively
- Download the build from this PR
- Install
- Open installed extension go onboarding and check out shadows in app

## Pre-Merge Checklist

- [x] PR template is filled out
- [x] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [x] PR is linked to the appropriate GitHub issue
- [x] PR has been added to the appropriate release Milestone

### + If there are functional changes:

- [x] Manual testing complete & passed
- [x] "Extension QA Board" label has been applied

### CODE REVIEW

File: ui/components/app/account-menu/index.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
   width: 320px;
   border-radius: 4px;
   background: var(--color-background-default);
-  box-shadow: rgba($black, 0.15) 0 2px 2px 2px;
+  box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
   min-width: 150px;
   color: var(--color-text-default);
```
Comment (georgewrmarshall) [inline ui/components/app/account-menu/index.scss:11]: ## Before

Screen Shot 2022-07-15 at 5 52 21 PMScreen Shot 2022-07-15 at 5 52 35 PM

## After

Screen Shot 2022-07-15 at 5 53 14 PM
Screen Shot 2022-07-15 at 5 53 30 PM

File: ui/components/app/gas-customization/gas-price-button-group/index.scss [hunk 2]
```diff
@@ -195,7 +195,7 @@
     color: var(--color-text-default);
     width: 108px;
     height: 97px;
-    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.151579);
+    box-shadow: var(--shadow-size-sm) var(--color-shadow-default);
     border-radius: 6px;
     border: none;
```
Comment (georgewrmarshall) [inline ui/components/app/gas-customization/gas-price-button-group/index.scss:201]: Don't think this component is used. Made a quick story to display that isn't in this commit

Screen Shot 2022-07-15 at 6 00 36 PM

File: ui/components/app/gas-customization/gas-slider/index.scss [hunk 3]
```diff
@@ -17,7 +17,7 @@
     height: 34px;
     width: 34px;
     background-color: var(--color-primary-default);
-    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.08);
+    box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
     border-radius: 50%;
     position: relative;
     z-index: 10;
```
Comment (georgewrmarshall) [inline ui/components/app/gas-customization/gas-slider/index.scss:23]: ## Before
Screen Shot 2022-07-15 at 6 06 25 PM

## After
Screen Shot 2022-07-15 at 6 04 23 PM

File: ui/components/app/home-notification/index.scss [hunk 4]
```diff
@@ -3,7 +3,7 @@
   flex-flow: column;
   justify-content: space-between;
   background: var(--color-background-default);
-  box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+  box-shadow: var(--shadow-size-md) var(--color-shadow-default);
   border: 1px solid var(--color-border-muted);
   border-radius: 8px;
   min-height: 116px;
```
Comment (georgewrmarshall) [inline ui/components/app/home-notification/index.scss:9]: ## Before
Screen Shot 2022-07-15 at 6 06 25 PM

## After
Screen Shot 2022-07-15 at 6 04 23 PM

**Review by Lojzicek — DISMISSED** [review]
Dekuji

File: ui/components/app/gas-customization/gas-slider/index.scss (hunk 3 above)
Comment (georgewrmarshall) [inline ui/components/app/gas-customization/gas-slider/index.scss:23]: Not sure this component is used and updating the shadow did nothing anyway. Going to remove

gas-slider-no-shadow

File: ui/components/app/modals/hide-token-confirmation-modal/index.scss [hunk 5]
```diff
@@ -2,7 +2,7 @@
   min-height: 250.72px;
   border-radius: 4px;
   background-color: var(--color-background-default);
-  box-shadow: 0 1px 7px 0 rgba(0, 0, 0, 0.5);
+  box-shadow: var(--shadow-size-md) var(--color-shadow-default);
```
Comment (georgewrmarshall) [inline ui/components/app/modals/hide-token-confirmation-modal/index.scss:6]: ### Before

hide-token-confirmation-modal before1
hide-token-confirmation-modal before2

### After

hide-token-confirmation-modal after1
hide-token-confirmation-modal after2

File: ui/components/app/modals/new-account-modal/index.scss [hunk 6]
```diff
@@ -3,7 +3,7 @@
   flex-flow: column nowrap;
   background-color: var(--color-background-default);
   border-radius: 10px;
-  box-shadow: 0 5px 16px rgba($black, 0.25);
+  box-shadow: var(--shadow-size-lg) var(--color-shadow-default);
```
Comment (georgewrmarshall) [inline ui/components/app/modals/new-account-modal/index.scss:7]: ## Before
new-account-modal-before-1
new-account-modal-before-2


### After
new-account-modal-after-2
new-account-modal after1

File: ui/components/ui/page-container/index.scss [hunk 7]
```diff
@@ -1,7 +1,7 @@
 .page-container {
   width: 408px;
   background-color: var(--color-background-default);
-  box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.08);
+  box-shadow: var(--shadow-size-sm) var(--color-shadow-default);
   z-index: 25;
   display: flex;
   flex-flow: column;
```
Comment (georgewrmarshall) [inline ui/components/ui/page-container/index.scss:7]: ## Before
page-container before1
page-container before2

## After
page-container after1
page-container after2

File: ui/components/app/signature-request-original/index.scss [hunk 8]
```diff
@@ -14,7 +13,6 @@
     @media screen and (max-width: $break-small) {
       width: 100%;
       top: 0;
-      box-shadow: none;
     }
 
     @media screen and (min-width: $break-large) {
```
Comment (georgewrmarshall) [inline ui/components/app/signature-request-original/index.scss:18]: ## Before

signature-request-original-before-1signature-request-original-before-2

## After
signature-request-original-after-1
signature-request-original-after-2

File: ui/components/app/signature-request/index.scss [hunk 9]
```diff
@@ -63,7 +62,7 @@
     font-size: 60px;
     color: var(--color-background-default);
     z-index: 1;
-    text-shadow: 0 4px 6px rgba(0, 0, 0, 0.422);
+    text-shadow: var(--shadow-size-sm) var(--color-shadow-default);
   }
 
   &__info {
```
Comment (georgewrmarshall) [inline ui/components/app/signature-request/index.scss:68]: ### Before
signature-request-original-before-1
signature-request-original-before-2

### After
signature-request after-1
signature-request after-2

File: ui/components/ui/button-group/index.scss [hunk 10]
```diff
@@ -61,7 +61,7 @@
     }
 
     &:hover {
-      box-shadow: 0 0 14px rgba(0, 0, 0, 0.18);
+      box-shadow: var(--shadow-size-sm) var(--color-shadow-default);
     };
   }
```
Comment (georgewrmarshall) [inline ui/components/ui/button-group/index.scss:67]: ### Before

button-group before1
button-group before2

### After
button-group after1
button-group after2

File: ui/components/ui/button/buttons.scss [hunk 11]
```diff
@@ -289,6 +309,10 @@ input[type="submit"][disabled] {
   &.btn-danger-primary {
     background-color: var(--color-error-default);
 
+    &:hover {
+      box-shadow: var(--shadow-size-sm) var(--color-error-shadow);
+    }
+
     &--disabled,
     &[disabled] {
       background-color: var(--color-error-disabled);
```
Comment (georgewrmarshall) [inline ui/components/ui/button/buttons.scss:318]: ### Before

button before1
button before2

### After

button after1
button after2

File: ui/components/ui/callout/callout.scss [hunk 12]
```diff
@@ -15,7 +15,7 @@
 
   &--dismissible {
     &#{$self}--first {
-      box-shadow: 0 -5px 5px -5px var(--color-overlay-default);
+      box-shadow: 0 -5px 5px -5px var(--color-shadow-default);
     }
   }
```
Comment (georgewrmarshall) [inline ui/components/ui/callout/callout.scss:21]: ### Before
callout before1
callout before2

### After
callout after1
callout after2

File: ui/components/ui/menu/menu.scss [hunk 13]
```diff
@@ -3,7 +3,7 @@
     @include H6;
 
     background: var(--color-background-default);
-    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.214);
+    box-shadow: var(--shadow-size-sm) var(--color-shadow-default);
     border-radius: 8px;
     width: 225px;
     color: var(--color-text-default);
```
Comment (georgewrmarshall) [inline ui/components/ui/menu/menu.scss:9]: ### Before

menu before1
menu before2

### After
menu after1
menu after2

File: ui/components/app/signature-request-original/index.scss (hunk 8 above)
Comment (georgewrmarshall) [inline ui/components/app/signature-request-original/index.scss:18]: ### Before

Screen Shot 2022-07-17 at 1 06 07 PM
Screen Shot 2022-07-17 at 1 06 30 PM

### After

signature-request-original after1
signature-request-original after2

File: ui/components/app/signature-request/index.scss (hunk 9 above)
Comment (georgewrmarshall) [inline ui/components/app/signature-request/index.scss:68]: ## Before
Screen Shot 2022-07-17 at 1 08 23 PM
Screen Shot 2022-07-17 at 1 08 42 PM


## After
Screen Shot 2022-07-17 at 1 10 59 PM
Screen Shot 2022-07-17 at 1 11 20 PM

File: ui/components/ui/page-container/index.scss [hunk 14]
```diff
@@ -1,7 +1,7 @@
 .page-container {
   width: 408px;
   background-color: var(--color-background-default);
-  box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.08);
+  box-shadow: var(--shadow-size-sm) var(--color-shadow-default);
   z-index: 25;
```
Comment (georgewrmarshall) [inline ui/components/ui/page-container/index.scss:5]: ### Before
Screen Shot 2022-07-17 at 1 17 51 PM
Screen Shot 2022-07-17 at 1 18 14 PM

## After
Screen Shot 2022-07-17 at 1 12 52 PM
Screen Shot 2022-07-17 at 1 13 20 PM

File: ui/components/ui/popover/index.scss [hunk 15]
```diff
@@ -104,6 +104,6 @@
     background: var(--color-background-default);
     position: absolute;
     transform: rotate(45deg);
-    box-shadow: 0 4px 30px rgba(0, 0, 0, 0.25);
+    box-shadow: var(--shadow-size-lg) var(--color-shadow-default);
   }
 }
```
Comment (georgewrmarshall) [inline ui/components/ui/popover/index.scss:109]: ## Before

Screen Shot 2022-07-17 at 1 29 05 PM
Screen Shot 2022-07-17 at 1 28 42 PM

## After
Screen Shot 2022-07-17 at 1 30 27 PM
Screen Shot 2022-07-17 at 1 30 46 PM

File: ui/components/ui/sender-to-recipient/index.scss [hunk 16]
```diff
@@ -107,7 +107,7 @@
         justify-content: center;
         flex: 1;
         border-radius: 4px;
-        box-shadow: 0 2px 4px rgba(0, 0, 0, 0.08);
+        box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
         padding: 6px;
         background: var(--color-background-default);
         cursor: pointer;
```
Comment (georgewrmarshall) [inline ui/components/ui/sender-to-recipient/index.scss:113]: ## Before
Screen Shot 2022-07-17 at 1 39 29 PM
Screen Shot 2022-07-17 at 1 39 59 PM

## After
Screen Shot 2022-07-17 at 1 38 18 PM
Screen Shot 2022-07-17 at 1 38 01 PM

File: ui/components/ui/tooltip/index.scss [hunk 17]
```diff
@@ -2,7 +2,7 @@
   .tippy-tooltip.tippy-tooltip--mm-custom-theme {
     background: var(--color-background-default);
     color: var(--color-text-default);
-    box-shadow: 0 0 14px rgba(0, 0, 0, 0.18);
+    box-shadow: var(--shadow-size-md) var(--color-shadow-default);
     padding: 12px 16px;
     padding-bottom: 11px;
```
Comment (georgewrmarshall) [inline ui/components/ui/tooltip/index.scss:8]: ## Before
Screen Shot 2022-07-17 at 1 41 52 PM
Screen Shot 2022-07-17 at 1 42 17 PM

## After
Screen Shot 2022-07-17 at 1 43 24 PM
Screen Shot 2022-07-17 at 1 43 42 PM

File: ui/components/ui/button-group/index.scss (hunk 10 above)
Comment (SaraCheikh) [reply ui/components/ui/button-group/index.scss:67]: this shadow in SM looks good, I'm curious to know if we are applying it only in hover state or by default?

**Review by Akatori-Design — COMMENTED** [review]
What exactly is the update I should be reviewing?

File: ui/components/ui/sender-to-recipient/index.scss (hunk 16 above)
Comment (SaraCheikh) [reply ui/components/ui/sender-to-recipient/index.scss:113]: Is the shadow only applied in dark mode here? for light mode it looks like the before and after have the same values

**Comment by SaraCheikh** [comment]
I can't see the new values applied, but they look better in the "after" in both themes. My doubt is how we are applying the shadows. In some modals we are using LG and other MD, and for some menus are using XS and other SM

**Comment by georgewrmarshall** [comment]
@Akatori-Design just looking for you to review the "Before" and "After" screenshots at each change to make sure I'm using the shadows correctly. @SaraCheikh has made some suggestions so I'll go through and update them to fit with out guidelines a bit better. Will get you to review once they are updated

File: ui/components/ui/url-icon/index.scss [hunk 18]
```diff
@@ -4,7 +4,7 @@
   background-position: center;
   border-radius: 50%;
   background-color: var(--color-background-default);
-  box-shadow: 0 2px 4px 0 rgba($black, 0.24);
+  box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
   flex: 0 0 auto;
   -moz-animation: fadein 1s;
```
Comment (georgewrmarshall) [inline ui/components/ui/url-icon/index.scss:10]: ## Before
url-icon before1
url-icon before2

## After
url-icon after1
url-icon after2

File: ui/css/itcss/components/send.scss [hunk 19]
```diff
@@ -864,7 +864,7 @@
     border: 1px solid var(--color-border-muted);
     border-radius: 4px;
     background-color: var(--color-background-default);
-    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.14);
+    box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
     display: flex;
     flex-flow: column;
```
Comment (georgewrmarshall) [inline ui/css/itcss/components/send.scss:870]: Pretty sure this is all unused CSS that should be removed. Will create a new PR and remove it.

File: ui/css/itcss/components/newui-sections.scss [hunk 20]
```diff
@@ -63,23 +63,23 @@ $sub-mid-size-breakpoint-range: "screen and (min-width: #{$break-large}) and (ma
...
 }
 
 @media screen and (min-width: 1281px) {
   .main-container {
     width: 62vw;
     min-height: 82vh;
-    box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.08);
+    box-shadow: var(--shadow-size-sm) var(--color-shadow-default);
   }
 }
```
Comment (georgewrmarshall) [inline ui/css/itcss/components/newui-sections.scss:85]: ## Before

Screen Shot 2022-07-19 at 2 34 46 PM
Screen Shot 2022-07-19 at 2 35 45 PM

## After

Screen Shot 2022-07-19 at 2 53 38 PM
Screen Shot 2022-07-19 at 2 54 00 PM

File: ui/pages/confirm-decrypt-message/confirm-decrypt-message.scss [hunk 21]
```diff
@@ -3,7 +3,7 @@
     width: 380px;
     border-radius: 8px;
     background-color: var(--color-background-default);
-    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.08);
+    box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
     display: flex;
     flex-flow: column nowrap;
     z-index: 25;
```
Comment (georgewrmarshall) [inline ui/pages/confirm-decrypt-message/confirm-decrypt-message.scss:9]: ## After

Screen Shot 2022-07-19 at 3 03 32 PM

File: ui/pages/confirm-encryption-public-key/confirm-encryption-public-key.scss [hunk 22]
```diff
@@ -3,7 +3,7 @@
     width: 380px;
     border-radius: 8px;
     background-color: var(--color-background-default);
-    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.08);
+    box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
     display: flex;
     flex-flow: column nowrap;
     z-index: 25;
```
Comment (georgewrmarshall) [inline ui/pages/confirm-encryption-public-key/confirm-encryption-public-key.scss:9]: ## After

Screen Shot 2022-07-19 at 3 05 12 PM
Screen Shot 2022-07-19 at 3 05 32 PM

File: ui/pages/confirmation/components/confirmation-network-switch/index.scss [hunk 23]
```diff
@@ -18,7 +18,7 @@
     align-items: center;
     justify-content: center;
     position: absolute;
-    filter: drop-shadow(0 1px 5px rgba(0, 0, 0, 0.25));
+    filter: drop-shadow(0 1px 5px var(--color-shadow-default));
 
     &::before {
       content: '';
```
Comment (georgewrmarshall) [inline ui/pages/confirmation/components/confirmation-network-switch/index.scss:24]: ## Before
confirmation-network-switch before1
confirmation-network-switch before2

## After
confirmation-network-switch after1
confirmation-network-switch after2

Comment (georgewrmarshall) [inline ui/pages/confirmation/components/confirmation-network-switch/index.scss:24]: ## Before

Screen Shot 2022-07-19 at 3 20 01 PM
Screen Shot 2022-07-19 at 3 20 12 PM

## After

Screen Shot 2022-07-19 at 3 18 40 PM
Screen Shot 2022-07-19 at 3 18 52 PM

File: ui/pages/create-account/index.scss [hunk 24]
```diff
@@ -4,7 +4,7 @@
 .new-account {
   width: 375px;
   background-color: var(--color-background-default);
-  box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.08);
+  box-shadow: var(--shadow-size-sm) var(--color-shadow-default);
   z-index: 25;
   height: unset;
   overflow: auto;
```
Comment (georgewrmarshall) [inline ui/pages/create-account/index.scss:10]: ## Before

Screen Shot 2022-07-19 at 3 24 33 PM
Screen Shot 2022-07-19 at 3 24 51 PM

## After

Screen Shot 2022-07-19 at 3 23 36 PM
Screen Shot 2022-07-19 at 3 24 09 PM

File: ui/pages/first-time-flow/seed-phrase/confirm-seed-phrase/index.scss [hunk 25]
```diff
@@ -85,7 +85,7 @@
       display: inline-flex;
 
       &:hover {
-        box-shadow: 0 2px 4px rgba(0, 0, 0, 0.25);
+        box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
       }
     }
```
Comment (georgewrmarshall) [inline ui/pages/first-time-flow/seed-phrase/confirm-seed-phrase/index.scss:91]: This isn't used anymore. What this looks like:
Screen Shot 2022-07-19 at 3 28 23 PM

Updated UI
Screen Shot 2022-07-19 at 3 29 10 PM

File: ui/pages/home/index.scss [hunk 26]
```diff
@@ -116,7 +116,7 @@
     position: absolute;
     top: 73px;
     opacity: 1;
-    box-shadow: 0 0 0 9999px rgba(0, 0, 0, 0.2);
+    box-shadow: var(--shadow-size-lg) var(--color-shadow-default);
     background: none;
   }
```
Comment (georgewrmarshall) [inline ui/pages/home/index.scss:122]: ## Before

Screen Shot 2022-07-19 at 3 32 32 PM
Screen Shot 2022-07-19 at 3 32 07 PM

## After
Screen Shot 2022-07-19 at 3 33 33 PM
Screen Shot 2022-07-19 at 3 33 08 PM

File: ui/pages/import-token/token-list/index.scss [hunk 27]
```diff
@@ -46,7 +46,7 @@
     background-position: center;
     border-radius: 50%;
     background-color: var(--color-background-default);
-    box-shadow: 0 2px 4px 0 rgba($black, 0.24);
+    box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
     margin-right: 12px;
     flex: 0 0 auto;
   }
```
Comment (georgewrmarshall) [inline ui/pages/import-token/token-list/index.scss:52]: ## Before

Screen Shot 2022-07-19 at 4 44 34 PM
Screen Shot 2022-07-19 at 4 44 49 PM


## After
Screen Shot 2022-07-19 at 4 46 48 PM

File: ui/pages/onboarding-flow/recovery-phrase/index.scss [hunk 28]
```diff
@@ -179,7 +179,7 @@
 
     &--with-input {
       width: 120px;
-      box-shadow: 0 3px 4px -3px rgba(0, 0, 0, 0.15);
+      box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
       border-width: 2px;
       border-radius: 13px;
       height: 32px;
```
Comment (georgewrmarshall) [inline ui/pages/onboarding-flow/recovery-phrase/index.scss:185]: ## After 

Screen Shot 2022-07-19 at 4 55 20 PM

File: ui/pages/settings/index.scss [hunk 29]
```diff
@@ -70,7 +70,7 @@
       &__list {
         background: var(--color-background-default);
         box-sizing: border-box;
-        box-shadow: 0 0 14px rgba(0, 0, 0, 0.18);
+        box-shadow: var(--shadow-size-md) var(--color-shadow-default);
         border-radius: 6px;
         position: absolute;
         width: 100%;
```
Comment (georgewrmarshall) [inline ui/pages/settings/index.scss:76]: ## Before
Screen Shot 2022-07-19 at 6 07 42 PM
Screen Shot 2022-07-19 at 6 07 29 PM

## After
Screen Shot 2022-07-19 at 5 00 15 PM
Screen Shot 2022-07-19 at 5 00 30 PM

File: ui/pages/swaps/build-quote/index.scss [hunk 30]
```diff
@@ -110,7 +110,7 @@
   &__open-dropdown,
   &__open-to-dropdown {
     max-height: 330px;
-    box-shadow: 0 0 14px rgba(0, 0, 0, 0.18);
+    box-shadow: var(--shadow-size-md) var(--color-shadow-default);
     position: absolute;
     width: 100%;
   }
```
Comment (georgewrmarshall) [inline ui/pages/swaps/build-quote/index.scss:116]: ## Before

Screen Shot 2022-07-19 at 6 09 05 PM
Screen Shot 2022-07-19 at 6 09 22 PM

## After

Screen Shot 2022-07-19 at 6 23 43 PM
Screen Shot 2022-07-19 at 6 23 56 PM

File: ui/pages/swaps/index.scss [hunk 31]
```diff
@@ -50,7 +50,7 @@
       width: 460px;
       background: var(--color-background-default);
       border: 1px solid var(--color-border-muted);
-      box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.08);
+      box-shadow: var(--shadow-size-sm) var(--color-shadow-default);
       border-radius: 8px;
       height: 620px;
     }
```
Comment (georgewrmarshall) [inline ui/pages/swaps/index.scss:56]: ## Before
Screen Shot 2022-07-20 at 8 13 46 AM
Screen Shot 2022-07-20 at 8 14 10 AM

## After
Screen Shot 2022-07-20 at 8 23 13 AM
Screen Shot 2022-07-20 at 8 23 36 AM

File: ui/pages/swaps/select-quote-popover/index.scss [hunk 32]
```diff
@@ -98,7 +98,7 @@
     &--selected {
       color: var(--color-primary-inverse);
       background: linear-gradient(90deg, var(--color-primary-default) 0%, var(--color-primary-alternative) 101.32%);
-      box-shadow: 0 10px 39px rgba(3, 125, 214, 0.15);
+      box-shadow: var(--shadow-size-lg) var(--color-shadow-default);
       border-radius: 8px;
       height: 64px;
```
Comment (georgewrmarshall) [inline ui/pages/swaps/select-quote-popover/index.scss:104]: ## Before
Screen Shot 2022-07-20 at 8 27 06 AM
Screen Shot 2022-07-20 at 8 32 28 AM

## After
Screen Shot 2022-07-20 at 8 24 56 AM
Screen Shot 2022-07-20 at 8 25 10 AM

File: .storybook/test-data.js [hunk 33]
```diff
@@ -1456,7 +1466,16 @@ const state = {
...
+          token: {
+            address: '0xaD6D458402F60fD3Bd25163575031ACDce07538D',
+            symbol: 'DAI',
+            decimals: 18,
+          },
+          history: {},
+        },
       },
       previousModalState: {
         name: null,
```
Comment (georgewrmarshall) [inline .storybook/test-data.js:1481]: Linting and adding some props to display a modal

File: package.json [hunk 34]
```diff
@@ -118,7 +118,7 @@
     "@material-ui/core": "^4.11.0",
     "@metamask/contract-metadata": "^1.31.0",
     "@metamask/controllers": "^30.0.2",
-    "@metamask/design-tokens": "^1.6.5",
+    "@metamask/design-tokens": "^1.8.0",
     "@metamask/eth-ledger-bridge-keyring": "^0.13.0",
     "@metamask/eth-token-tracker": "^4.0.0",
     "@metamask/etherscan-link": "^2.1.0",
```
Comment (georgewrmarshall) [inline package.json:124]: Updating design tokens to v1.8 because it includes the shadow tokens

File: ui/components/app/account-menu/index.scss [hunk 35]
```diff
@@ -5,7 +5,7 @@
   width: 320px;
   border-radius: 4px;
   background: var(--color-background-default);
-  box-shadow: rgba($black, 0.15) 0 2px 2px 2px;
+  box-shadow: var(--shadow-size-sm) var(--color-shadow-default);
   min-width: 150px;
   color: var(--color-text-default);
```
Comment (georgewrmarshall) [inline ui/components/app/account-menu/index.scss:11]: ## Before
Screen Shot 2022-07-20 at 9 11 02 AMScreen Shot 2022-07-20 at 9 11 20 AM

## After
Screen Shot 2022-07-20 at 9 11 50 AMScreen Shot 2022-07-20 at 9 12 11 AM

File: ui/components/app/dropdowns/dropdown.js [hunk 36]
```diff
@@ -18,7 +18,7 @@ export class Dropdown extends Component {
       borderRadius: '4px',
       padding: '8px 16px',
       background: 'var(--color-background-default)',
-      boxShadow: 'rgba(0, 0, 0, 0.15) 0px 2px 2px 2px',
+      boxShadow: 'var(--shadow-size-sm) var(--color-shadow-default)',
       ...innerStyle,
     };
```
Comment (georgewrmarshall) [inline ui/components/app/dropdowns/dropdown.js:24]: This component is used in the network dropdown

## Before
Screen Shot 2022-07-20 at 9 15 52 AM
Screen Shot 2022-07-20 at 9 16 11 AM

## After
Screen Shot 2022-07-20 at 9 16 45 AM
Screen Shot 2022-07-20 at 9 16 26 AM

File: ui/components/app/modals/hide-token-confirmation-modal/hide-token-confirmation.stories.js [hunk 37]
```diff
@@ -0,0 +1,11 @@
...
+import HideTokenConfirmationModal from '.';
+
+export default {
+  title: 'Components/UI/HideTokenConfirmationModal', // title should follow the folder structure location of the component. Don't use spaces.
+  id: __filename,
+};
+
+export const DefaultStory = () => <HideTokenConfirmationModal />;
+
+DefaultStory.storyName = 'Default';
```
Comment (georgewrmarshall) [inline ui/components/app/modals/hide-token-confirmation-modal/hide-token-confirmation.stories.js:11]: Adding missing story to help display changes

File: ui/components/app/modals/new-account-modal/new-account-modal.stories.js [hunk 38]
```diff
@@ -0,0 +1,11 @@
...
+import NewAccountModal from './new-account-modal.component';
+
+export default {
+  title: 'Components/UI/NewAccountModal', // title should follow the folder structure location of the component. Don't use spaces.
+  id: __filename,
+};
+
+export const DefaultStory = () => <NewAccountModal />;
+
+DefaultStory.storyName = 'Default';
```
Comment (georgewrmarshall) [inline ui/components/app/modals/new-account-modal/new-account-modal.stories.js:11]: Adding new story to help display changes

File: ui/components/app/signature-request/index.scss [hunk 39]
```diff
@@ -53,7 +53,6 @@
     border-radius: 50%;
     border: 1px solid var(--color-background-default);
     position: absolute;
-    box-shadow: 0 2px 2px 0.5px rgba(0, 0, 0, 0.19);
```
Comment (georgewrmarshall) [inline ui/components/app/signature-request/index.scss:56]: Removing unnecessary shadow from identicon border.

File: ui/components/ui/slider/slider.component.js [hunk 40]
```diff
@@ -35,21 +35,21 @@ const styles = {
...
       height: 22,
       width: 22,
       marginTop: -8,
       marginLeft: -8,
       border: 'none',
-      boxShadow: '0px 0px 14px 0px rgba(0, 0, 0, 0.18)',
+      boxShadow: 'var(--shadow-size-md) var(--color-shadow-default)',
     },
   },
 };
```
Comment (georgewrmarshall) [inline ui/components/ui/slider/slider.component.js:55]: ## Before
Screen Shot 2022-07-20 at 9 47 29 AM
Screen Shot 2022-07-20 at 9 47 47 AM

## After
Screen Shot 2022-07-20 at 9 40 25 AM
Screen Shot 2022-07-20 at 9 40 45 AM

File: ui/pages/permissions-connect/redirect/index.scss [hunk 41]
```diff
@@ -39,7 +39,6 @@
     align-items: center;
     justify-content: center;
     position: absolute;
-    filter: drop-shadow(0 1px 5px rgba(0, 0, 0, 0.25));
```
Comment (georgewrmarshall) [inline ui/pages/permissions-connect/redirect/index.scss:42]: removing unnecessary shadow

File: ui/components/app/gas-customization/gas-slider/gas-slider.stories.js [hunk 42]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    min: {
+      control: 'number',
+    },
+  },
+};
+
+export const DefaultStory = () => <GasSlider />;
+
+DefaultStory.storyName = 'Default';
```
Comment (georgewrmarshall) [inline ui/components/app/gas-customization/gas-slider/gas-slider.stories.js:34]: Adding missing story to help display changes

Screen Shot 2022-07-20 at 9 56 13 AM

**Review by SaraCheikh — APPROVED** [review]

**Review by Akatori-Design — APPROVED** [review]

**Review by garrettbear — APPROVED** [review]

**Pull request merged**